nc-news-bh-final-vite/src/components/DeleteComment.jsx

```diff
@@ -0,0 +1,26 @@
+import {useEffect, useState} from 'react'
+import { deleteComment } from '../utils/api.js'
+
+
+const DeleteComment = ({comment_id, setIsDeleting})=>{
+
+        function handleClick(){
+            console.log("yo")
+            console.log(comment_id)
+            setIsDeleting(true)
+            deleteComment(comment_id).then((data)=>{
+                setIsDeleting(false)
+            })
```

Comment:
Need a catch block here and display a message if the post request is unsuccessful.